dbschema/role.gel

```diff
@@ -22,5 +22,6 @@ module default {
     RegionalDirector,
     StaffMember,
     Translator,
+    MultiplicationFinanceApprover,
```

Comment:
Order matters for postgres, and positioning this new member above others shifts their indexes. PG is not happy with this shift as we use these member values in "views" / access policies.
So adding to bottom prevents shifting existing members that are already in use.

I think there's no problem with having it out of order here.
We can take care to try to order alphabetically in queries if/when needed.

Reply:
Not a big deal, but this is unfortunate.